hypothesis-python/docs/usage.rst

```diff
@@ -2,10 +2,14 @@
 Open Source Projects using Hypothesis
 =====================================
 
-The following is a non-exhaustive list of open source projects I know are using Hypothesis. If you're aware of
-any others please add them to the list! The only inclusion criterion right now is that if it's a Python library
+The following is a non-exhaustive list of open source projects I know are
```

Comment:
Maybe switch "I" to "we"?

I wonder if this list is even useful given libraries.io?

Reply:
I'm basically just going for incremental fixes here, with a larger overhaul on the horizon.  Specifically I'm working on #1193 this weekend (hopefully including auto-gen API reference) and probably a short tutorial by late May for pycon.sg, which covers the easy half of #774.